src/test/scala/kanaloa/reactive/dispatcher/queue/QueueWithBackPressureSpec.scala

```diff
@@ -64,71 +64,76 @@ class QueueWithBackPressureSpec extends SpecWithActorSystem {
 
   "isOverCapacity" >> {
 
-    val q = TestActorRef[QueueWithBackPressure](Queue.withBackPressure(BackPressureSettings(
-      maxBufferSize = 10,
-      thresholdForExpectedWaitTime = 5.minutes
-    ))).underlyingActor
+    val q = TestActorRef[QueueWithBackPressure](Queue.withBackPressure(
+      BufferHistorySettings(),
+      BackPressureSettings(
+        maxBufferSize = 10,
+        thresholdForExpectedWaitTime = 5.minutes
+      )
+    )).underlyingActor
 
-    def status(ps: (Int, Int, LocalDateTime)*): QueueStatus = QueueStatus(bufferHistory = ps.map { case (dispatched, size, time) ⇒ BufferHistoryEntry(dispatched, size, time) }.toVector)
+    def status(ps: (Int, Int, Int, LocalDateTime)*): QueueStatus = QueueStatus(bufferHistory = ps.map { case (dispatched, queueSize, workerSize, time) ⇒ BufferHistoryEntry(dispatched, queueSize, workerSize, time) }.toVector)
+
+    def statusWithQueueSize(ps: (Int, LocalDateTime)*): QueueStatus = status(ps.map { case (queueSize, time) ⇒ (1, queueSize, 1, time) }: _*)
```

Comment:
`statusWithQueueSize` and `status` are kinda hard to read, could be split into multiple lines